backend/src/database/seeders/20220713115148-seed-users.js

```diff
@@ -1,9 +1,12 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const bcrypt = require('bcrypt');
+
 module.exports = {
-  up(queryInterface) {
+  async up(queryInterface) {
```

Comment:
I don't believe this needs to be an async method since you are only returning a Promise and not using `await` inside it

Reply:
yes, it is using await for `bcrypt.hash`